x-pack/plugin/gpu/src/main/java/org/elasticsearch/xpack/gpu/codec/ES92GpuHnswVectorsWriter.java

```diff
@@ -205,6 +209,8 @@ private void flushFieldsWithoutMemoryMappedFile(Sorter.DocMap sortMap) throws IO
                     cuVSResourceManager.release(cuVSResources);
                 }
             }
+            var elapsed = started - System.nanoTime();
+            logger.debug("Flushed [{}] vectors in [{}ms]", numVectors, elapsed / 1e-6);
```

Comment:
Did you mean to multiply by 1e-6?

May be instead of 1e-6, we can do elapsed / 1_000_000.0;

Reply:
🤦 Yes!